xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/uicomponents/suggest/suggest.js

```diff
@@ -730,7 +730,7 @@ var XWiki = (function(XWiki){
         noHighlight: true, // we do the highlighting ourselves
         containerTagName: 'a'
       });
-      item.containerElement.setAttribute('href', arr[i].url);
+      item.containerElement.setAttribute('href', arr[i].url || 'javascript:void(0)');
```

Comment:
Maybe add a comment explaining why the `'javascript:void(0)'` alternative is required for empty urls.

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/2238/commits/bca95f38b237e50d19004759e8abd6e06b4f6996